scripts/build-dependencies.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+
+function findNode() {
+    return $(which npm 1>/dev/null 2>&1)
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --no-color)
+    NOCOLOR=true
+    ;;
+    *) # If it wasn't --no-color, we assume it was the path of the root of the project
+    SUPPLIED_APP_FOLDER_PATH=$key
+    ;;
+esac
+shift
+done
+
+MYPATH=$( cd $SUPPLIED_APP_FOLDER_PATH && pwd )
+ROOT=$MYPATH/..
+EXTRA_NPM_ARGS=""
+
+if [ "$MYPATH" = "" ] ; then
+    echo "Build directory not specified, exiting."
+    exit 1
+fi
+
+if [ "$NOCOLOR" = true ]; then
+    EXTRA_NPM_ARGS="--no-color $EXTRA_NPM_ARGS"
+fi
+
+echo "MYPATH=$MYPATH"
+echo "ROOT=$ROOT"
+
+if [ ! -d $MYPATH/app-www/js/build ] ; then
+    mkdir $MYPATH/app-www/js/build
```

Comment:
Should this have the `-p` flag? (recursively create the directories if they don't exist)